core/src/main/java/org/apache/carbondata/core/constants/CarbonLoadOptionConstants.java

```diff
@@ -71,7 +71,7 @@
   public static final String CARBON_OPTIONS_TIMESTAMPFORMAT =
           "carbon.options.timestamp.format";
 
-  public static final String CARBON_OPTIONS_TIMESTAMPFORMAT_DEFAULT = "";
+  public static final String CARBON_OPTIONS_TIMESTAMPFORMAT_DEFAULT = "yyyy-MM-dd HH:mm:ss";
```

Comment:
It's better use CarbonCommonConstants.CARBON_TIMESTAMP_DEFAULT_FORMAT as default value.

Reply:
hi @zzcclp, mean this?
```
CARBON_OPTIONS_TIMESTAMPFORMAT_DEFAULT = CarbonCommonConstants.CARBON_TIMESTAMP_DEFAULT_FORMAT
```

Reply:
yeah